webapp/channels/src/components/admin_console/team_channel_settings/group/group_list.tsx

```diff
@@ -40,7 +40,7 @@ const Header = () => {
     );
 };
 
-interface Props {
+export interface Props {
```

Comment:
So here the Props consists of both the passed in component props and props coming from the redux connect. Can you please help seperate the OwnProps.